HiddenStateExtractor/vq_vae_supp.py

```diff
@@ -145,14 +145,16 @@ def prepare_dataset_v2(dat_fs,
     dataset = np.stack([tensors[key] for key in ts_keys], 0)
     return dataset, ts_keys
 
-def reorder_with_trajectories(dataset, relations, seed=None):
+def reorder_with_trajectories(dataset, relations, seed=None, w_a=1.1, w_t=0.1):
```

Comment:
Hi @miaecle , do you call this function only after first 100 epochs to 'burn-in' the model without matching loss? Is the corresponding training loop in this PR?

Reply:
Actually no, this is called before all training.